rustup-init.sh

```diff
@@ -493,15 +493,27 @@ check_help_for() {
 
     case "$_arch" in
 
-        # If we're running on OS-X, older than 10.13, then we always
-        # fail to find these options to force fallback
         *darwin*)
         if check_cmd sw_vers; then
-            if [ "$(sw_vers -productVersion | cut -d. -f2)" -lt 13 ]; then
-                # Older than 10.13
-                echo "Warning: Detected OS X platform older than 10.13"
-                return 1
-            fi
+            case $(sw_vers -productVersion) in
+                10.*)
+                    # If we're running on OS-X, older than 10.13, then we always
+                    # fail to find these options to force fallback
+                    if [ "$(sw_vers -productVersion | cut -d. -f2)" -lt 13 ]; then
```

Comment:
The internal pain I have that this shell command is run multiple times is silly, but I'll state it regardless.

Reply:
Given the cut is different each time, I figured I'd just rerun it.  If this were a real programming language I'd have split it into parts once.  But sw_vers should be pretty fast to run, and consistent through a rustup-init or the user has worse problems :D

Reply:
Oh, it’s definitely not a real problem, especially considering that this should only be run like twice ever.